Source/Menu/MainForm.cs

```diff
@@ -231,21 +231,19 @@ orderby tool.Text
                 var path = dir + @"\Documentation\";
                 if (Directory.Exists(path))
                 {
-                    foreach (string entry in Directory.GetFiles(path))
+                    // Load English documents
+                    LoadDocuments(docs, path, null);
```

Comment:
```suggestion
                    LoadDocuments(docs, path);
```
Simplify by making the `code` parameter optional.